extensions/2.0/Khronos/KHR_animation2/README.md

```diff
@@ -0,0 +1,187 @@
+# KHR\_animation2
+
+## Contributors
+
+* Alexey Knyazev
+* Ben Houston, Threekit
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Don McCurdy
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Joe Herdman, [@jbherdman](https://github.com/jbherdman)
+* Lewis Weaver, Microsoft
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright (C) 2018-2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is based on the animation features of glTF 2.0. The structure of the schemas stay the same.  
+  
+The only major addition is, that the output values are mapped using a [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901).
+  
+### Motivation
+
+At this point of time, one can only target the transformation or weight data of a node.  
+With this extension, one can technically target any value (`scalar`, `vec2`, `vec3`, `vec4` and scalars in an `array`) in glTF e.g.
+
+* Color factors in materials
+* Camera field of view
+
+Even using a JSON pointer, the targets and their expected behavior are explictly defined.
+
+Current and future extensions have to write against this specification to allow to animate specific values.
+
+#### Current and future restrictions
+
+The calculated values can be out of range of the minimum and maximum value, however the values have to be clamped using the properties min and max value before setting the value.
+
+##### Not animatable properties
+
+`glTFid`
```

Comment:
On further thought, I think in some cases it would actually be interesting to animate `glTFid`, but would still exclude it here:

- stop-motion like effects, where a mesh is switched to another one during a frame
  - workaround: scaling of nodes, like now
- image sprite effects, where a texture is swapped out to another one during animation
  - workaround: texture sheet + UV animation which is reasonable

Reply:
+1 for omitting `glTFid`, I'm worried it would be wildly complex to support in such an open-ended way. Consider: adding/removing lights means all materials must recompile, some engines must pre-compile materials, and it's impossible to predict which animations might play at the same time. 

The variant-based extensions are a good alternative for these cases.

Reply:
That's the reason I tried to define the expclicit list what is allowed to be animated.